src/Config/SDK/ComponentProvider/OpenTelemetrySdk.php

```diff
@@ -431,6 +617,67 @@ private function getLoggerProviderConfig(ComponentProviderRegistry $registry, No
                     ->end()
                 ->end()
                 ->append($registry->componentList('processors', LogRecordProcessorInterface::class))
+                ->arrayNode('logger_configurator/development')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->arrayNode('default_config')
+                            ->addDefaultsIfNotSet()
+                            ->children()
+                                ->booleanNode('disabled')->defaultFalse()->end()
+                            ->end()
+                        ->end()
+                        ->arrayNode('loggers')
+                            ->arrayPrototype()
+                                ->children()
+                                    ->scalarNode('name')->isRequired()->cannotBeEmpty()->end()
+                                    ->arrayNode('config')
+                                        ->addDefaultsIfNotSet()
+                                        ->children()
+                                            ->booleanNode('disabled')->defaultFalse()->end()
+                                        ->end()
+                                    ->end()
+                                ->end()
+                            ->end()
+                        ->end()
+                    ->end()
+                ->end()
+            ->end()
+        ;
+
+        return $node;
+    }
+
+    private function getPropagatorConfig(ComponentProviderRegistry $registry, NodeBuilder $builder): ArrayNodeDefinition
+    {
+        $node = $builder->arrayNode('propagator');
+        $node
+            ->beforeNormalization()
+                ->ifArray()
+                ->then(static function (array $value): array {
+                    $existing = [];
+                    foreach ($value['composite'] as $item) {
+                        $existing[] = key($item);
+                    }
+                    foreach (explode(',', $value['composite_list'] ?? '') as $name) {
+                        $name = trim($name);
+                        if ($name && !in_array($name, $existing)) {
+                            $value['composite'][][$name] = null;
```

Comment:
```suggestion
                            $value['composite'][][$name] = null;
                            $existing[] = $name;
```

Should add test with `composite_list: "tracecontext,tracecontext"`